tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -125,6 +125,24 @@ def test_alice_profiler_user_workflow_single_batch(
         ],
         include_citation=True,
     )
+
+    assert (
+        expectation_suite.expectations
+        == alice_columnar_table_single_batch["expected_expectation_suite"].expectations
+    )
+
+    """
+    Deleting some "parameter_builders" from both actual and expected configurations, because nested dictionaries are
```

Comment:
I'm not a huge fan of this, but understand that it is some acceptable tech debt given that to change it requires a larger fix to our serialization mechanisms.

Reply:
@anthonyburdi I kept trying for hours and could not make it work, even though they were equal as confirmed manually.  At least there is a full test in the VolumeDataAssistant test module.  Thank you.